layers/+spacemacs/spacemacs-defaults/packages.el

```diff
@@ -393,7 +393,8 @@
   (spacemacs|hide-lighter page-break-lines-mode))
 
 (defun spacemacs-defaults/init-process-menu ()
-  (evilified-state-evilify process-menu-mode process-menu-mode-map))
+  (evilified-state-evilify process-menu-mode process-menu-mode-map
```

Comment:
`evilified-state-evilify` is not recommended.
Use `evilified-state-evilify-map` instead.

Reply:
Do you want me to make the change in this PR?  I'm not familiar enough with the two macros to know the difference immediately, so I just made the minimal change (the existing code uses `evilified-state-evilify`).  Feel free to suggest changes or add commits as you see fit.